theme/llms-button.scss

```
@@ -0,0 +1,172 @@
+
+// External-link affordance — dim it; it's a signal, not a button.
+.rp-llms-view-options__external-icon {
+  opacity: 0.5;
+  width: 13px;
+  height: 13px;
+}
+
+// ─── Dark mode ───────────────────────────────────────────────────────────
+// On a near-black surface the rest border + wash both need a bit more
+// presence to read; mirrors the SubsiteRow dark-mode bump.
+//
```

Stylelint (17.12.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)